apps/statistics/flow-query-certificates-pdf-conf.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var db                = require('../../db')
+  , assign            = require('es5-ext/object/assign')
+  , dateToParamConfig = require('../../apps-common/query-conf/date-to');
+
+module.exports = [
+	require('../../apps-common/query-conf/date-from'),
+	assign({},
+		dateToParamConfig,
+		{
+			ensure: function (value) {
+				if (!value) return new db.Date();
+				return dateToParamConfig.ensure(value);
```

Comment:
Two things. The `dateTo` param doesn't have to be different, so yes we could use the same code as here in both cases to have default `dateTo`. As to the whole query handler, they do differ, as here we need to pass more params like `service` and `certficate` (in the regular view those two params are handled on the client, and are explicitly removed from server query). Here I would prefer to keep the filtering logic on the server, so that really the data that needs to be displayed is passed to pdf renderer.